tests/unit/test__signing.py

```diff
@@ -744,14 +746,31 @@ def test_conformance_client(test_data):
 
 @pytest.mark.parametrize("test_data", _BUCKET_TESTS)
 def test_conformance_bucket(test_data):
-    resource = "/{}".format(test_data["bucket"])
-    _run_conformance_test(resource, test_data)
+    if "urlStyle" in test_data and test_data["urlStyle"] == "BUCKET_BOUND_DOMAIN":
```

Comment:
Please update to latest release of conformance tests this value was updated to `BUCKET_BOUND_HOSTNAME`. Last nit.